scripts/world_bank/health_nutrition_population/population_get_mcf.py

```diff
@@ -0,0 +1,35 @@
+def get_mcf(series_lst):
+    '''
+    gets mcf by splitting description and using fstrings
+    '''
+    nodes = []
+    #don't  use df; automate
+    with open("output\World_bank_hnp_population.mcf", 'w', encoding = 'utf-8') as file:
+        print(end = '')
+        mcf = ''
+        for current_series in series_lst:
+            print('\n'+current_series)
+            statvars = ['Node', 'typeOf', 'description', 'populationType',
+                      'measuredProperty', 'gender', 'statType', 'age']
+            node, age, gender = '', '', ''
+            age = current_series[9:11]
+            gender = ''
+            if 'MA' in current_series:
+                gender = 'Male'
+            else:
+                gender = 'Female'
+            node = f'Count_Persons_{(int(age))}Years_{gender}'
+            desc = f'Age population, age {age}, {gender.lower()}, interpolated'
```

Comment:
why is it necessary to note that it is interpolated -- was it specially called out in the dataset?

Reply:
Yes - in the dataset, is was specified as interpolated